src/commons.ts

```diff
@@ -97,14 +96,12 @@ export default class Commons {
     }
 
     let uploadStopped: boolean = true;
-    Commons.extensionWatcher = chokidar.watch(this.en.ExtensionFolder, {
-      depth: 0,
-      ignoreInitial: true
-    });
-    Commons.configWatcher = chokidar.watch(this.en.PATH + "/User/", {
-      depth: 2,

```

Comment:
How are you handling the `depth` as 2 in File System Watcher

Reply:
You mean the watcher should not watch under directories in `workspaceStorage`.
OK, I'll try handling it! Thanks.

Reply:
@shanalikhan 
Fixed by https://github.com/shanalikhan/code-settings-sync/pull/763/commits/758baf7007fc66be3833243d20e927efba39f606
But it's not beautiful way... umm...